src/components/manage/Blocks/Image/View.test.jsx

```diff
@@ -4,11 +4,23 @@ import { render } from '@testing-library/react';
 import { MemoryRouter } from 'react-router-dom';
 import View from './View';
 
+import config from '@plone/volto/registry';
+
+config.settings.imageScales = {
+  large: 768,
+  preview: 400,
+  mini: 200,
+  thumb: 128,
+  tile: 64,
+  icon: 32,
+  listing: 16,
+};
+
 describe('Image View Component', () => {
   test('renders a view image component with a local image', () => {
     const { getByRole } = render(<View data={{ url: '/image.jpg' }} />);
     const img = getByRole('img');
-    expect(img).toHaveAttribute('src', '/image.jpg/@@images/image');
+    expect(img).toHaveAttribute('src', '/image.jpg/@@images/image/listing');
```

Comment:
`listing` scale by default? why is that? that scale is not in the plone.volto ones.